scripts/vagrant/multi/tunnel_vagrant.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
nit: I think this is identical to the single node tunnel script? Can we have both point to the same one?